js/components/user/list.vue

```diff
@@ -58,6 +58,13 @@ export default {
                 align: 'center',
                 type: 'metric',
                 width: 95
+            }, {
+                label: this._('Roles'),
+                key: 'roles',
+                sort: 'roles',
+                align: 'center',
+                type: 'metric',
```

Comment:
Hum, it's not a metric. This type is meant to change color given the value (and so expect a number).
You should use type `label`, a `label_type` function and maybe a `label_func` (see: https://github.com/opendatateam/udata/blob/dev/js/components/datatable/cells/label.vue)